lib/twitter_web/templates/browse/show.html.eex

```diff
@@ -0,0 +1,17 @@
+<h1>Show Public Post</h1>
+
+<ul>
+
+  <li>
+    <strong>body:</strong>
+    <%= @post.body %>
+  </li>
+  <li>
+    <strong>Likes: </strong>
+    <%= %>
```

Comment:
Maybe you could have a list of users who liked the post?
Could be achieved by adding some preloads.